### PR TITLE
Create referral-phish-emails.md

### PR DESCRIPTION
**Pull request by alimajalt**
Proposing accompanying email AHQ for review.

### CODE REVIEW

File: Email Queries/referral-phish-emails.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Referral infrastructure credential phishing emails
+The "Referral" infrastructure is a point-in-time set of infrastructure direclty associated with spoofed emails that imitate SharePoint and other legitimate products in order to conduct crednetial phishing. The operator is also known to use legitimate URL infrastructure such as Google, Microsoft and Digital Ocean to host their phishing pages.
```
Comment (DulceMontemayor) [inline Email Queries/referral-phish-emails.md:2]: Please correct typo error: 'direclty' should be 'directly'

File: Email Queries/referral-phish-emails.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## Query
+This query looks for instances where known malicious original senders that are approximately close to the word "Referral" associated with multiple phishing campaigns in multiple months in 2021. These mails also attempt to bypass protections and gain access to inboxes by spoofing the recipient domain in the displayed email address. This query will match instances where the displayed email address matches the recipient's domain and join to the email URL data for easy hunting on potential malicious credential theft sites.
```
Comment (DulceMontemayor) [inline Email Queries/referral-phish-emails.md:4]: I've updated the verbiage for clarity, pls check if technical context is intact, if yes, please replace with this:

'Use this query to search for instances of malicious senders associated with multiple phishing campaigns for a few months in 2021, with subjects approximately similar to "Referral". These mails also attempt to bypass protections and access inboxes by spoofing the recipient domain in the displayed email address. This query will match instances where the displayed email address matches the recipient's domain and join to the email URL data for easy hunting on potential malicious credential theft sites.'

File: Email Queries/referral-phish-emails.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+| where SenderMailFromDomain in (EmailAddresses)
+| extend RecipientDomain = extract("[^@]+$", 0, RecipientEmailAddress)
+| where SenderFromDomain == RecipientDomain
+| join EmailUrlInfo on $left.NetworkMessageId == $right.NetworkMessageId
+```
+
+
+## Category
+
+This query can be used to detect the following attack techniques and tactics ([see MITRE ATT&CK framework](https://attack.mitre.org/)) or security configuration states.
```
Comment (DulceMontemayor) [inline Email Queries/referral-phish-emails.md:21]: Pls replace with:

'Use this query to detect the following attack techniques and tactics ([see MITRE ATT&CK framework](https://attack.mitre.org/)) or security configuration states.'

**Review by DulceMontemayor — COMMENTED** [review]
Need some updates

**Review by alimajalt — COMMENTED** [review]
Looks good to me.

**Review by DulceMontemayor — APPROVED** [review]

**Pull request merged**